demo/AriesOpenAPIDemo.md

```diff
@@ -705,11 +705,11 @@ The following table lists endpoints that you need to call ("REST service") and c
 | Protocol Step        | Faber (Verifier)       | Alice (Holder/Prover)     | Notes |
 | -------------------- | ---------------------- | ------------------------- | ----- |
 | Send Proof Request | **`POST /present-proof-2.0/send-request`** | | REST service |
-| Receive Proof Request | | <agent_cb>/present_proof_v2_0 | callback |
+| Receive Proof Request | | <agent_cb>/present_proof_v2_0 | callback(webhook or websocket) |
```

Comment:
I think it makes sense to add `webhook`, but not sure about websocket.  The use of websockets or http is a transport level concern -- how the webhook is received -- not an alternative to the webhook.  It is always a webhook.